modules/ding_gatewayf/ding_gatewayf.module

```diff
@@ -241,6 +246,70 @@ function ding_gatewayf_callback() {
 }
 
 /**
+ * Do a check on the checksum given from gatewayf.
```

Comment:
Is there any official documentation we can link to about this security check? I mean: How do we know that this implementation is correct.